### ISSUE TITLE
Update render-fidelity "update-screenshots" CLI to have more options.

### ISSUE THREAD

**Issue by bhouston** [google/model-viewer#4536]
### Description

We need better CLI options for the update-screenshot CLI tool.

Specifically I would like to see:

- allow for picking one or more render engines, not just one.
- allow for picking any subset of scenarios via string matches.
- enable headless or not.  Because having Chrome pop up repeated is disruptive to local development.
- enable parallel renderer runs (excluding Stellar, and V-Ray), because it will be faster and there is amble CPU available for this.

Something like:
```bash
% npm run update-screenshots --renderer=filament --renderer=model-viewer --scenario=anisotropy --headless=true --parallel=true
```

Or something like this:
```
% npx update-screenshots -r filament -r model-viewer -s anisotropy -h -p
```

I was thinking of using commander for the CLI parsing as it is sort of industry standard.

**Comment by elalish** [comment]
Agreed, that would be a fantastic update!

**Comment by elalish** [comment]
Fixed by #4542, thanks!
